src/course-outline/outline-sidebar/OutlineSidebar.jsx

```diff
@@ -4,18 +4,16 @@ import { Hyperlink } from '@edx/paragon';
 import { useIntl } from '@edx/frontend-platform/i18n';
 
 import HelpSidebar from '../../generic/help-sidebar';
-import { useHelpUrls } from '../../help-urls/hooks';
 import { getFormattedSidebarMessages } from './utils';
 
-const OutlineSideBar = ({ courseId }) => {
+const OutlineSideBar = ({
+  courseId,
+  learnMoreVisibilityUrl,
```

Comment:
ref: remove as described in the comment above

Reply:
Done